recv.go

```diff
@@ -154,6 +156,12 @@ func (rcvr *Receiver) Run() {
 				if len(meterType) > 0 && !meterType[uint(pkt.MeterType())] {
 					continue
 				}
+				if *unique {
+					if lastValue[uint(pkt.MeterID())] == pkt.MeterValue() {
```

Comment:
This will fail if the meter hasn't been seen before and `pkt.MeterValue()` is 0.

Check for existence with:
```go
if val, ok := lastValue[pkt.MeterID()]; ok && val == pkt.MeterValue() {
    ...
}
```